packages/manager/apps/dedicated/client/app/dedicatedCloud/dedicatedCloud.bundle.js

```diff
@@ -109,7 +109,6 @@ import './mailing-list/subscribe/dedicatedCloud-mailing-list-subscribe.service';
 import './operation/dedicatedCloud-operation.controller';
 import './operation/dedicatedCloud-operation.routes';
 import './operation/executionDateEdit/dedicatedCloud-operation-executionDateEdit.controller';
-import './price/dedicatedCloud-price.controller';
```

Comment:
Is `packages/manager/apps/dedicated/client/app/dedicatedCloud/price/dedicatedCloud-price.html`file still needed?

Reply:
Stupid me ! I deleted all the files 🙂